bpf/bpf_lxc.c

```diff
@@ -848,46 +848,6 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx, __u32 *d
 	}
 #endif /* ENABLE_HOST_FIREWALL && !ENABLE_ROUTING */
 
-#ifdef ENABLE_EGRESS_GATEWAY
```

Comment:
Do we still have any egress gateway logic in `bpf_lxc.c`? If not, we can update `bpf/Makefile` and `bpf/complexity-tests/*/bpf_lxc.txt` a bit.